igor-web/src/main/groovy/com/netflix/spinnaker/igor/wercker/WerckerBuildMonitor.groovy

```diff
@@ -188,9 +183,13 @@ class WerckerBuildMonitor extends CommonPollingMonitor<PipelineDelta, PipelinePo
                     ))
         } catch (e) {
             log.error("Error processing runs for [{}:{}]", kv("master", master), kv("pipeline", pipeline), e)
-            if (e.cause instanceof RetrofitError) {
-                def re = (RetrofitError) e.cause
-                log.error("Error communicating with Wercker for [{}:{}]: {}", kv("master", master), kv("pipeline", pipeline), kv("url", re.url), re)
+            if (e instanceof SpinnakerServerException) {
+                def re = (SpinnakerServerException) e.getCause()
```

Comment:
The only use of `re` in the old code was to get the url.  That's now available from [SpinnakerHttpException](https://github.com/spinnaker/kork/blob/825c81a177d5c3b11a286d664759ea2786a4bd87/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java#L213-L215), so I don't think it makes sense to look at `e.getCause()` anymore.